### ISSUE TITLE
Report act from react-test-renderer

### ISSUE THREAD

**Issue by Belco90** [testing-library/eslint-plugin-testing-library#494]
### What rule do you want to change?

no-unnecessary-act

### Does this change cause the rule to produce more or fewer warnings?

More warnings

### How will the change be implemented?

As discussed in https://github.com/testing-library/eslint-plugin-testing-library/discussions/488 the `act` method from `react-test-renderer` must be reported too.

### Example code

```js
import { act } from "react-test-renderer";
import { fireEvent } from "@testing-library/react-native";

act(() => {
     ...fireEvent..
})
```

### How does the current rule affect the code?

`act` is not reported since it's imported from `react-test-renderer`

### How will the new rule affect the code?

`act` imported from `react-test-renderer` should be reported too.

### Do you want to submit a pull request to change the rule?

Yes
